alephium/contracts/sequence/unexecuted_sequence.ral

```diff
@@ -1,11 +1,11 @@
-Contract UndoneSequence(
+Contract UnExecutedSequence(
```

Comment:
Discussion: it seems `unexecuted` is valid English word, we could use `UnexecutedSequence` for contract name and `unexecutedSequence` for variables.

@h0ngcha0 What's your thoughts? I am not good at English.

Reply:
Sounds good :+1: